flutter/lib/src/integrations/flutter_framework_feature_flag_integration.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:sentry/sentry.dart';
+
+const _featureFlag = 'FLUTTER_ENABLED_FEATURE_FLAGS';
+
+// The Flutter framework feature flag works like this:
+// An enabled (experimental) feature gets added to the `FLUTTER_ENABLED_FEATURE_FLAGS`
+// dart define. Being in there means the feature is enabled, the feature is disabled
+// if it's not in there.
+// As a result, we also don't know the whole list of flags, but only the active ones.
+//
+// See https://github.com/flutter/flutter/pull/168437
+class FlutterFrameworkFeatureFlagIntegration
+    extends Integration<SentryOptions> {
+  final String flags;
+
+  FlutterFrameworkFeatureFlagIntegration({
+    @visibleForTesting this.flags = const String.fromEnvironment(_featureFlag),
+  });
+
+  @override
+  FutureOr<void> call(Hub hub, SentryOptions options) {
+    final enabledFeatureFlags = flags.split(',');
+
+    for (final featureFlag in enabledFeatureFlags) {
+      Sentry.addFeatureFlag(featureFlag, true);
+    }
+    options.sdk.addIntegration('FlutterFrameworkFeatureFlagIntegration');
```

Comment:
```suggestion
    options.sdk.addIntegration('FlutterFrameworkFeatureFlag');
```

I wanna omit the integration from now in the sdk integration string, I'll also update other integrations to match this

Reply:
I guess it makes sense to align the naming scheme then too. Right now it's not always `CamelCase` I believe.

Reply:
yep I agree, not so urgent because these are mainly used for our internal dashboards but I'll see to it that we'll align here